openapi-v1.yaml

```diff
@@ -5298,6 +5298,47 @@ paths:
           schema:
             $ref: "#/definitions/Error"
 
+  /notebooks/{namespace}/upload:
+    parameters:
+      - type: string
+        name: namespace
+        in: path
+        required: true
+        description: The namespace of the notebook
+      - name: X-TILEDB-CLOUD-ACCESS-CREDENTIALS-NAME
+        in: header
+        description: Optional registered access credentials to use for creation
+        type: string
+        required: false
+    post:
```

Comment:
According to the [docs](https://swagger.io/docs/specification/2-0/file-upload/)

We need to add multipart/form-data

````
consumes:
  - multipart/form-data

Reply:
👍